podman/api/version.py

```diff
@@ -1,3 +1,4 @@
 """Version of PodmanPy."""
-__version__ = "3.0.0"
+
+__version__ = "3.2.0"
```

Comment:
This is the second 3.2.0 I've seen in here.  Can we put it in once place and include it?

Reply:
Ooops, nvm, the first was a Makefile.